utils/CMakeLists.txt

```diff
@@ -3,17 +3,17 @@ file(GLOB UTILS_SOURCES "*.c")
 # see https://cmake.org/Wiki/CMake_RPATH_handling
 
 # use, i.e. don't skip the full RPATH for the build tree
-SET(CMAKE_SKIP_BUILD_RPATH  FALSE)
+set(CMAKE_SKIP_BUILD_RPATH  FALSE)
 
 # when building, don't use the install RPATH already
 # (but later on when installing)
-SET(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
+set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
 
-SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
+set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
```

Comment:
this seems suspect

Reply:
@mpreisler Why? I think it needs to be there to link the installed `oscap` utility with correct `libopenscap.so` library. If you already have installed `libopenscap.so` from an RPM package and you build and install OpenSCAP from source to a custom path, you want `oscap` linked with your built `libopenscap.so` library and not with the distribution library.

Reply:
The `/lib` parth is hardcoded. This needs to be lib64 on 64bit. Either way, use a variable, don't construct paths hardcoded, that's never the way to go. If you use variables the user can also customize all of the paths.